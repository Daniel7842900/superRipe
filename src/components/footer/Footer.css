/* footer style */

/* Common style */
footer {
  height: 20vh;
  background-color: #e36b81;
}

.contact-us-div {
  font-size: 1.5rem;
  text-align: center;
}

.contact-us-div p {
  margin: auto;
  padding: 0.5rem 0;
}

.contact-us-div p a:first-child {
  color: blue;
  text-decoration: none;
}

/* Extra small devices (phones) */
@media only screen and (max-width: 446px) {
  .contact-us-div {
    font-size: 1.75em;
    text-align: center;
  }

  .contact-us-div p {
    padding: 0.75rem 0;
  }
}

/* Medium devices (landscape tablets) */
@media only screen and (min-width: 768px) {
  .contact-us-div {
    font-size: 1.5em;
    text-align: center;
  }
}

/* Large devices (laptops/desktops) */
@media only screen and (min-width: 992px) {
  .contact-us-div {
    font-size: 2rem;
    text-align: center;
  }

  .contact-us-div p {
    padding: 0.5rem 0;
  }
}

/* Extra large devices (large laptops/desktops) */
@media only screen and (min-width: 1200px) {
  .contact-us-div {
    font-size: 2.25rem;
    text-align: center;
  }

  .contact-us-div p {
    padding: 0.5rem 0;
  }
}

/* End - footer style */
