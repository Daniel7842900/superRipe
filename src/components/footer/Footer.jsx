import React from "react";
import "./Footer.css";

const Footer = () => {
  return (
    <footer>
      <div className="contact-us-div">
        <p>Contact us about our app at</p>
        <p>
          <a href="">superripe_info@gmail.com</a>
        </p>
      </div>
    </footer>
  );
};

export default Footer;
