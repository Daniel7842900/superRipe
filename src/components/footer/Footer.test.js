import Footer from "./Footer";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("<Footer />", () => {
  it("renders without crashing", () => {
    shallow(<Footer />);
  });

  it("matches snapshot", () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
