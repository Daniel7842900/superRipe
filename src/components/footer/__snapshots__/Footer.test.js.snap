// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> matches snapshot 1`] = `
<footer>
  <div
    className="contact-us-div"
  >
    <p>
      Contact us about our app at
    </p>
    <p>
      <a
        href=""
      >
        superripe_info@gmail.com
      </a>
    </p>
  </div>
</footer>
`;
